ci/prow/e2e-crd-variable

```diff
@@ -2,8 +2,8 @@
 
 set -euxo pipefail
 
-check_kmm_mod () {
-oc debug node/$NODE -T -- chroot /host sh -c "lsmod | grep kmm"
+check_kmm_kmod () {
+  oc debug node/$NODE -T -- chroot /host sh -c "lsmod | grep kmm"
```

Comment:
Can we extract it to `check-kmm-kmod` script and source it everywhere instead of duplicating it?